hack/boilerplate.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
we need to use the k8s one: https://github.com/kubernetes-sigs/kueue/blob/ee2a9a2634489c7ad449b390d8640aa7a739258e/main.go#L1-L15

Reply:
ditto all other places

Reply:
Updated the copyrights